docs/docs/zh/guide/datasource/dameng.md

```diff
@@ -0,0 +1,17 @@
+# DAMENG数据源
+
+![dameng](../../../../img/new_ui/dev/datasource/dameng_cn.png)
```

Comment:
Please use English screenshot, we will not maintain screenshots in different languages.

Reply:
Ok, I will take a screenshot in English